systemd/ua.service

```diff
@@ -0,0 +1,20 @@
+# If you are uninterested in the (free for personal use) Ubuntu Advantage
+# services, including security updates after standard EOL and kernel patching
+# without rebooting, then you can safely stop and disable this service:
+# sudo systemctl stop ua.service
+# sudo systemctl disable ua.service
+
+[Unit]
+Description=Ubuntu Advantage Daemon
+Documentation=man:ubuntu-advantage https://github.com/canonical/ubuntu-advantage-client/blob/main/README.md https://ubuntu.com/advantage
```

Comment:
Could pointing to the upstream, development version of the README file cause any sort of confusion for users throughout the development cycles?

Reply:
Good point. Let's leave it out for now. Ideally at some point in the future we would have better documentation, maybe with version specific links. But for now just pointing to the readme on `main` could likely be confusing